virus.py

```diff
@@ -0,0 +1,101 @@
+import datajoint as dj
+
+schema = dj.schema('virus', locals())
+
+@schema
+class ConstructSource(dj.Lookup):
+    # source of the construct
+    definition = """
+    construct_source : varchar(64) # source of the construct
+    """
+    contents = [['Addgene'], ['Constomly made']]
+
+@schema
+class Construct(dj.Manual):
+    # information of the construct
+    definition = """
+    construct_name : varchar(255)    # name of a construct
+    ---
+    -> ConstructSource
+    construct_notes = null : varchar(255)   # notes of the construct
+    """
+
+@schema
+class GeneElementType(dj.Lookup):
+    # type of the gene element type
```

Comment:
this comment is not too helpful esp given the comment on the `type` field. Also, the attribute name `type` may be a bit too generic - perhaps something like `gene_element_type` or `ge_type` or something similar?